lib/QubitXmlImport.class.php

```diff
@@ -819,7 +819,24 @@ public static function replaceLineBreaks($node)
       }
       else
       {
-        $nodeValue .= preg_replace('/[\n\r\s]+/', ' ', $child->nodeValue);
+        if($child->tagName=='extent') 
```

Comment:
Some spaces missing around the operator and after the if, it should be ...

`if ($child->tagName == 'extent')`